modules/exploits/multi/misc/weblogic_deserialize_asyncresponseservice.rb

```diff
@@ -9,62 +9,76 @@ class MetasploitModule < Msf::Exploit::Remote
+          to execute code on the vulnerable host.
+        },
+        'Author' => [
+          'Andres Rodriguez - 2Secure (@acamro) <acamro[at]gmail.com>', # Metasploit Module
         ],
-      'License' => MSF_LICENSE,
-      'References' =>
-        [
-          ['CVE', '2017-10271'],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2019-2725'],
           ['CNVD-C', '2019-48814'],
           ['URL', 'http://www.cnvd.org.cn/webinfo/show/4999'],
           ['URL', 'https://www.oracle.com/technetwork/security-advisory/alert-cve-2019-2725-5466295.html'],
           ['URL', 'https://twitter.com/F5Labs/status/1120822404568244224']
         ],
-      'Privileged' => false,
-      'Platform' => %w{ unix win solaris },
-      'Targets' =>
-        [
-          [ 'Unix',
-            'Platform' => 'unix',
-            'Arch' => ARCH_CMD,
-            'DefaultOptions' => {'PAYLOAD' => 'cmd/unix/reverse_bash'}
+        'Privileged' => false,
+        'Platform' => %w[unix win solaris],
+        'Targets' => [
+          [
+            'Unix',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_bash' }
+            }
           ],
-          [ 'Windows',
-            'Platform' => 'win',
-            'Arch' => [ARCH_X64, ARCH_X86],
-            'DefaultOptions' => {'PAYLOAD' => 'windows/meterpreter/reverse_tcp'}
+          [
+            'Windows',
+            {
+              'Platform' => 'win',
+              'Arch' => [ARCH_X64, ARCH_X86],
+              'DefaultOptions' => { 'PAYLOAD' => 'windows/meterpreter/reverse_tcp' }
+            }
           ],
-          [ 'Solaris',
-            'Platform' => 'solaris',
-            'Arch' => ARCH_CMD,
-            'DefaultOptions' => {'PAYLOAD' => 'cmd/unix/reverse_perl'},
-            'Payload' => {
-              'Space'       => 2048,
-              'DisableNops' => true,
-              'Compat'      =>
+          [
+            'Solaris',
+            {
+              'Platform' => 'solaris',
+              'Arch' => ARCH_CMD,
+              'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_perl' },
+              'Payload' => {
+                'Space' => 2048,
+                'DisableNops' => true,
+                'Compat' =>
                 {
                   'PayloadType' => 'cmd',
-                  'RequiredCmd' => 'generic perl telnet',
+                  'RequiredCmd' => 'generic perl telnet'
                 }
+              }
             }
           ]
         ],
-      'DefaultTarget' => 0,
-      'DefaultOptions' =>
-        {
+        'DefaultTarget' => 0,
+        'DefaultOptions' => {
           'WfsDelay' => 12
         },
-      'DisclosureDate' => '2019-04-23'))
+        'DisclosureDate' => '2019-04-23',
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE ],
+          'SideEffects' => [ ARTIFACTS_ON_DISK, IOC_IN_LOGS ],
```

Comment:
Looks like `ARTIFACTS_ON_DISK` can be removed. We typically see that used when there is a command stager which is not in use here. Both targets execute an OS command through the SOAP request. IOC_IN_LOGS sounds reasonable though given the request may be logged.

Reply:
done!